cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -1,7 +1,7 @@
-{% set heapster_memory = "300Mi" -%}
+{% set heapster_memory = "100Mi" -%}
```

Comment:
heh... if we think that 100 might be too small, I definitely vote for something bigger than 150.
Maybe 250?